tests/components/google_assistant/__init__.py

```diff
@@ -221,7 +221,7 @@ def should_2fa(self, state):
             "action.devices.traits.TransportControl",
             "action.devices.traits.MediaState",
         ],
-        "type": "action.devices.types.SETTOP",
+        "type": "action.devices.types.TV",
```

Comment:
`_attr_device_class` was being ignored in `DemoTVShowPlayer`, so I've had to adjust the tests now that it is fixed.